evennia/contrib/base_systems/character_creator/README.md

```diff
@@ -0,0 +1,87 @@
+# Character Creator contrib
+by InspectorCaracal
+
+This contrib is designed to be used in MULTISESSION_MODE = 2 or higher, where characters are not automatically created to match the account. To use this with lower modes, you'll need to implement your own solution for preventing the built-in automatic character creation.
+
+## Installation
+
+In your game folder `commands/default_cmdsets.py`, import and add `ContribCmdCharCreate` to your `AccountCmdSet`.
+
+Example:
+```py
+from evennia.contrib.base_systems.character_creator.character_creator import ContribCmdCharCreate
+
+class AccountCmdSet(default_cmds.AccountCmdSet):
+    
+    def at_cmdset_creation(self):
+        super().at_cmdset_creation()
+        self.add(ContribCmdCharCreate)
+```
+
+In your game folder `typeclasses/accounts.py`, import and inherit from `ContribChargenAccount` on your Account class.
+
+(Alternatively, you can copy the `at_look` method directly into your own class.)
+
+Example:
+```py
+from evennia.contrib.base_systems.character_creator.character_creator import ContribChargenAccount
+
+class Account(ContribChargenAccount):
+    # your Account class code
+```
+
+Lastly, in your `settings.py` file, define `CHARGEN_MENU` to your character creation menu module's location.
+
+Example:
+```py
+CHARGEN_MENU = "evennia.contrib.base_systems.character_creator.example_menu"
+```
```

Comment:
Since this is a setting only introduced for this contrib, I would suggest making this setting optional, and have the code fall back to the contrib-example if nothing is given. This makes the install shorter and easier to test out. Something like

----

By default, the new `charcreate` command provided by this contrib will use the char-creation menu found in `evennia.contrib.base_systems.character_creator.example_menu`. Look at this to get an idea for how it's set up. To tell the contrib code to use your own menu, point to it with the `CHARGEN_MENU` setting. So if your custom chargen menu nodes are found in `mygame/word/chargen_menu.py`, you'd add: 

    CHARGEN_MENU = "world.chargen_menu"